src/components/NbtscanTool/NbtscanTool.tsx

```diff
@@ -141,7 +145,6 @@ const NbtscanTool = () => {
         setAllowSave(false);
     }, [setOutput]);
 
-    //define user interface of the tool
     return (
         <form onSubmit={form.onSubmit(onSubmit)}>
```

Comment:
This build fail because in your interface you have declare promise such as DumpPacket but in your form you have not include it in a value that we pass thru. I see that you have created a usestate instead for your dumppacket which is fine but you will need to remove the interface promise in order to pass the build.